tests/test_create.py

```diff
@@ -432,3 +432,40 @@ def test_create_cooler_from_dask():
         #     pixels,
         #     ordered=False
         # )
+
+
+@pytest.mark.parametrize(
+    "fp", [op.join(datadir, "hg19.GM12878-MboI.matrix.2000kb.cool")]
+)
+def test_create_scool(fp):
+    outfile_scool = tempfile.NamedTemporaryFile(suffix='.scool', delete=False)
```

Comment:
Could you replace using a tempfile with the `isolated_filesystem` context manager as above?